plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.java

```diff
@@ -20,6 +20,8 @@
     public static String OCBinaryLocationIncompatibleErrorMessage;
     public static String OCBinaryLocationDontExistsErrorMessage;
     public static String NoOCBinaryLocationErrorMessage;
+    public static String couldNoSyncPodsToFolder;
```

Comment:
this is long running issue. The only thing I'd agree in is that we should be consistent, either extract all or no messages at all. To me it is very unlikely (and never happened in all the years that I am part of the project) that there will be any other language than english. Thus extracting messages brings no benefit but additional indirection. IMHO the best thing to do is to have things prepared (like using NLS.bind) but **NOT** extracting strings.

Reply:
in Fuse Tooling we were asked to do it continuously to be ready "in case of", if you are going in the opposite way in JBoss Tools, our work is useless :-(